server/controllers/friendController.ts

```diff
@@ -0,0 +1,48 @@
+import { RequestHandler } from "express";
+
+import { FriendService } from "../services";
+
+class UserController {
+    static addNewFriend: RequestHandler = async (req, res) => {
+        try {
+            await FriendService.addNewFriend(req.params.userId, req.user._id);
+
+            res.json({ message: "New friend added", success: true });
+        } catch (err) {
+            res.status(400).json({ message: err.messsage, success: false });
```

Comment:
Do you have any specific error handler? If no maybe better to create global error handler?

Reply:
I'll think about it